tools/sigclient/pkg/query/query.go

```diff
@@ -734,17 +734,17 @@ func RunQueryFromFile(dest string, numIterations int, prefix string, continuous,
 							finalHits = eValue
 							hits, err = utils.VerifyInequality(finalHits, relation, expectedValue)
 						case map[string]interface{}:
-							for k, v := range eValue {
-								if k == "value" {
-									var ok bool
-									finalHits, ok = v.(float64)
-									if !ok {
-										log.Fatalf("RunQueryFromFile: Returned total matched is not a float: %v", v)
-									}
-									hits, err = utils.VerifyInequality(finalHits, relation, expectedValue)
-
-								}
+							value := eValue["value"]
+							receivedRelation := eValue["relation"]
```

Comment:
```suggestion
							value, ok := eValue["value"]
							receivedRelation, ok := eValue["relation"]
```
And check the `ok` values

Reply:
addressed in the latest commit.